### ISSUE TITLE
Setting graphic.interactive = true, gives errors

### ISSUE THREAD

**Issue by mmmintdesign** [pixijs-userland/graphics-smooth#28]
Setting graphic.interactive = true, gives errors
![image](https://user-images.githubusercontent.com/76491577/196248409-b3d7b1ff-5db1-489c-b268-8f6039ebe1b4.png)

**Comment by mmmintdesign** [comment]
> That's double-pixi error. graphics-smooth minimal requirement is 6.0.4 and you put something like 6.0.1 in packagejson , right? If not, anyway, remove your packagelock node_modules and install everything again.

The version is at 6.5.6. I've tried multiple times to delete/reinstall node_modules and package.json. 
Not sure if I'm doing something wrong. I'll try it on another computer.

**Comment by mmmintdesign** [comment]
Not sure what's the best way to share a demo, but here is repo: https://github.com/mmmintdesign/pixi-demo

**Comment by ivanpopelyshev** [comment]
I looked at the demo and I dont understand whats wrong with interaction. @bigtimebuddy can you please look at it?

**Comment by bigtimebuddy** [comment]
This worked fine for me: https://codesandbox.io/s/eager-napier-88ev5c

I think this is a Webpack issue.

**Comment by ivanpopelyshev** [comment]
Now I remember! Yes, it can be webpack issue, there was one not long ago. Please update webpack to latest.

**Comment by bigtimebuddy** [comment]
Obviously, use whatever tools meet your needs, but I think Parcel is more usable out-of-the-box and faster. Would highly recommend over Webpack.

**Comment by mmmintdesign** [comment]
Updated Webpack and all related packages, still getting the bug.

Just used to Webpack, might have to switch to Parcel

**Comment by ivanpopelyshev** [comment]
Ugh, those bundlers, ughhhhh
